samples/sample_binary.d

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Run this file with the `dlang_sample.sh` script.
+
+import std.stdio;
+import MyGame.Sample;
+import std.conv;
+
+// Example how to use FlatBuffers to create and read binary buffers.
+void main()
+{
+    auto builder = new FlatBufferBuilder(512);
+
+    // Create a string for our Monster as name.
+    auto name = builder.createString("MyMonster");
+
+    // Create some weapons for our Monster 
+    uint[] weaps;
+    foreach (i; 0 .. 3)
+    {
+        weaps ~= Weapon.createWeapon(builder,
+            builder.createString("Weapon." ~ i.to!string()), cast(short) i); // Use the `createWeapon()` helper function to create the weapons, since we set every field.
```

Comment:
Move the comment to the previous line, instead of creating very wide line.

Also bad alignment.